test/components/dragBoxLayerTests.ts

```diff
@@ -741,21 +680,24 @@ describe("Layer Components", () => {
           { x: midPoint.x, y: midPoint.y },
           { x: midPoint.x, y: midPoint.y + dragDistance }
         );
-        let bounds = dbl.bounds();
+        const bounds = dbl.bounds();
         assert.strictEqual(bounds.topLeft.x, midPoint.x, "new box was started at the drag start position (x)");
         assert.strictEqual(bounds.topLeft.y, midPoint.y, "new box was started at the drag start position (y)");
+        assert.strictEqual(bounds.bottomRight.x, midPoint.x, "new box was ended at the drag end position (x)");
+        assert.strictEqual(bounds.bottomRight.y, midPoint.y + dragDistance, "new box was ended at the drag end position (y)");
         svg.remove();
       });
 
-      it("does not have movable CSS classe when enabled(false) or movable(false)", () => {
-        assert.isFalse(dbl.hasClass("movable"), "initially does not have \"movable\" CSS class");
+      it("has movable CSS classes only when enabled and movable", () => {
+        const movableClass = "movable";
```

Comment:
Looks like we should probably factor this out as a `static` variable on `DragBoxLayer` -- todo later.